definitions/video-conviva/conviva-dashboard.json

```diff
@@ -0,0 +1,435 @@
+{
+  "name": "Conviva",
+  "description": null,
+  "pages": [
+    {
+      "name": "By Device Name",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Attempts",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 2,
+            "height": 4
+          },
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT sum(`conviva.attempts`) as 'Attempts' FROM Metric since 381 minutes ago until 20 minutes ago FACET device_name limit max"
```

Comment:
since and until in the dashboards will not allow the users to use the time picker.

So would be best to remove them